policy/modules/services/xserver.if

```diff
@@ -187,9 +187,6 @@ template(`xserver_role',`
 		allow $2 xserver_tmpfs_t:file rw_file_perms;
 	')
 
-	# XCB Event Queue: used by the Qt library for example
-	allow $2 xserver_tmp_t:file rw_file_perms;
```

Comment:
You've confirmed this is no longer required for Qt 6?

Reply:
Neither of xserver_tmp_t and xserver_tmpfs_t are needed in the latest Qt version 6.

But I have only carried out testing with the qtdoc examples git://code.qt.io/qt/qtdoc.git because all real world applications that I use are still linked to Qt5.